gridstatus/tests/base_test_iso.py

```diff
@@ -232,7 +244,6 @@ def test_get_load_today(self):
             or (df["Interval Start"].dt.date == today).all()
             or (df["Interval End"].dt.date == today).all()
         )
-        return df
```

Comment:
I removed this because pytest warns that tests should not return a value, and this behavior will be deprecated in the future. The only place that used this was in `test_pjm` which I've updated.